share/smack/top.py

```diff
@@ -296,6 +297,15 @@ def smack_headers():
 def smack_lib():
   return os.path.join(smack_root(), 'share', 'smack', 'lib')
 
+def objc_clang_compile_command(args, lib = False):
+  cmd = default_clang_compile_command(args,lib)
+  if sys.platform in ['linux', 'linux2']:
+    objc_flags = try_command(['gnustep-config', '--objc-flags'])
+    cmd += objc_flags.split()
+  elif sys.platform == 'darwin'
+    pass # add future macOS modifications here.
```

Comment:
Instead of `pass`, you should do `sys.exit` here with an appropriate message. Search for `exit` in this file for an example.